<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutwindow</class>
 <widget class="QDialog" name="aboutwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QLabel" name="program_title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>[Program title]</string>
   </property>
  </widget>
  <widget class="QLabel" name="version_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[Version]</string>
   </property>
  </widget>
  <widget class="QLabel" name="license_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[License]</string>
   </property>
  </widget>
  <widget class="QLabel" name="webpage_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[Webpage]</string>
   </property>
  </widget>
  <widget class="QLabel" name="webpage_link_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>[Link to webpage]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="license_description_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>[License description]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="developers_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[Developers]</string>
   </property>
  </widget>
  <widget class="QLabel" name="assets_used_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[Assets used]</string>
   </property>
  </widget>
  <widget class="QLabel" name="assets_used_list_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>[Assets used list]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="asset_rights_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[Assets right]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
